infra/cifuzz/cifuzz.py

```diff
@@ -33,6 +33,35 @@
 import repo_manager
 import utils
 
+# From clusterfuzz: src/python/crash_analysis/crash_analyzer.py
```

Comment:
@jonathanmetzman are we sure we want this complexity? IMO having a fuzzing log (which ends with the stacktrace) + test case should be sufficient. Given that the log is an artifact people would download and open locally, I see no problem if there are some log lines in the beginning.

Reply:
I sorta agree (I think abhishek asked for it in a file) but I can see why its better UX to post stacktrace seperately.
The GitHub interface for viewing logs is pretty bad and buggy. Maybe instead of just a file we can output the trace in a seperate step.

Reply:
@jonathanmetzman  From my experience with GitHub actions output we don't want the users to have to get information from the logs.  You can not even copy a text segment from it, so I am a proponent of surfacing via text files. But do you think the file should contain the full logs or just the stacktrace?